cmd/zoekt-webserver/main.go

```diff
@@ -123,6 +123,7 @@ func main() {
 	index := flag.String("index", build.DefaultDir, "set index directory to use")
 	html := flag.Bool("html", true, "enable HTML interface")
 	enableRPC := flag.Bool("rpc", false, "enable go/net RPC")
+	enableStream := flag.Bool("stream", false, "enable streamSearch")
```

Comment:
lets just enableStream if we enableRPC. This is an extension we added so this avoids the need for us to update docker images/etc.